lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Parameters/EvaluatorHeartbeatPeriodInMs.cs

```diff
@@ -0,0 +1,28 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Common.Runtime.Evaluator.Parameters
+{
+      [NamedParameter(Documentation = "Evaluator Heartbeat Period in ms", ShortName = "EvaluatorHeartbeatPeriodInMs", DefaultValue = "4000")]
+    public class EvaluatorHeartbeatPeriodInMs : Name<int>
```

Comment:
I can change it to internal now. One day if client wants to set it, we might have to either make it public or create some configuration module for it.